usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -195,29 +194,17 @@ def test_cluster_import_valid(valid_session_credentials, valid_trusted_pool):
                 with data ``{"message": "Accepted"}``.
 
         """
-    nodes = api.get_nodes()
-    node_ids = None
-    for cluster in nodes["clusters"]:
-        if cluster["sds_name"] == "gluster":
-            node_ids = cluster["node_ids"]
-            break
-    node_fqdns = []
-    msg = "`sds_pkg_name` of node {} should be `gluster`, it is {}"
-    for node in nodes["nodes"]:
-        if node["node_id"] in node_ids:
-            pytest.check(node["detectedcluster"]["sds_pkg_name"] == "gluster",
-                         msg.format(node["fqdn"],
-                         node["detectedcluster"]["sds_pkg_name"]))
-            node_fqdns.append(node["fqdn"])
-    node_ids = [x["node_id"] for x in nodes["nodes"]
-                if x["fqdn"] in valid_trusted_pool]
+    clusters = api.get_cluster_list()
```

Comment:
Please change this code to include only nodes from *valid_trusted_pool* otherwise this code will add all available nodes.